packages/auto-approve/src/process-checks/owl-bot-template-changes.ts

```diff
@@ -63,10 +66,11 @@ export class OwlBotTemplateChanges extends Process implements LanguageRule {
       this.incomingPR.author
     );
 
-    const titleMatches = checkTitleOrBody(
-      this.incomingPR.title,
-      this.classRule.titleRegex
-    );
+    const titleMatches =
+      // We don't want it to include fix, feat, or !
+      !checkTitleOrBody(this.incomingPR.title, this.classRule.titleRegex[0]) &&
+      // We do want it to include [autoapprove] in title
+      checkTitleOrBody(this.incomingPR.title, this.classRule.titleRegex[1]);
```

Comment:
Rather than relying on the position of the regex for exclusion, can we add a new config for excludeTitleRegex or something like that?